index.html

```diff
@@ -25,9 +25,223 @@
             <input type="checkbox" class="checkbox" value="log-in">
             <p class="left">Keep me logged in</p>
             <a href="https://www.facebook.com/login/identify?ctx=recover"><p class="right">Forgot your password?</p></a>
+
           </div>
         </form>  
       </div>   
     </nav>
+    <section>
+      <div class="main-section">
+        <div class="left-wrapper">
+          <div>
+            <h1>Connect with your friends and the world around you on Facebook.</h1>
+          </div>
+          <div class="top">
+            <img src="section-newspaper.png">
```

Comment:
make sure you give all your images alts. e.g. `<img src="golfball.png" alt="Image of a Golfball">